rlberry/manager/evaluation.py

```diff
@@ -226,27 +230,13 @@ def read_writer_data(data_source, tag=None, preprocess_func=None, id_agent=None)
         tags = None
 
     writer_datas = []
-    if input_dir is not None:
-        if take_last_date:
-            subdirs = list((Path(input_dir) / "manager_data").iterdir())
-            agent_name_list = [str(p.stem).split("_")[0] for p in subdirs]
-            for name in agent_name_list:
-                filename, dir_name = _get_last_xp(input_dir, name)
-                writer_datas.append(_load_data(filename, dir_name, id_agent))
-        else:
-            agent_name_list = [str(Path(p).stem).split("_")[0] for p in input_dir]
-            agent_dirs = [str(Path(p).parent).split("_")[0] for p in input_dir]
-
-            for id_f, filename in enumerate(input_dir):
-                writer_datas.append(_load_data(filename, agent_dirs[id_f], id_agent))
-    else:
-        for manager in experiment_manager_list:
-            # Important: since manager can be a RemoteExperimentManager,
-            # it is important to avoid repeated accesses to its methods and properties.
-            # That is why writer_data is taken from the manager instance only in
-            # the line below.
-            writer_datas.append(manager.get_writer_data())
-        agent_name_list = [manager.agent_name for manager in experiment_manager_list]
+    for manager in experiment_manager_list:
+        # Important: since manager can be a RemoteExperimentManager,
+        # it is important to avoid repeated accesses to its methods and properties.
+        # That is why writer_data is taken from the manager instance only in
+        # the line below.
```

Comment:
This is not true anymore, delete this comment maybe to be more clear.

Reply:
done !